q2_fmt/plugin_setup.py

```diff
@@ -327,6 +328,38 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.sample_pprs,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata, 'time_column': Str,
+                'baseline_timepoint': Str, 'subject_column': Str,
+                'filter_missing_references': Bool,
+                'drop_incomplete_subjects': Bool,
+                'drop_incomplete_timepoints': List[Str]},
+    outputs=[('pprs_dists', Dist1D[Ordered, Matched] % Properties("pprs"))],
+    input_descriptions={
+        'table': peds_table},
+    parameter_descriptions={
+        'metadata': metadata,
+        'time_column': time_column,
+        'baseline_timepoint': baseline_timepoint,
+        'subject_column': subject_column,
+        'filter_missing_references': filter_missing_references,
+        'drop_incomplete_subjects': drop_incomplete_subjects,
+        'drop_incomplete_timepoints': drop_incomplete_timepoints
+    },
+    output_descriptions={
+        'pprs_dists': 'The distributions for the PPRS measure, grouped by'
+                      ' the selected `time_column`. Also contains the'
+                      ' numerator and denominator for PPRS calulations.'
+    },
+    name='',
+    description='',
+    citations=[citations['aggarwala_precise_2021']],
+    examples={}
```

Comment:
Do you want to add in your PeePRS usage example here?

Reply:
love that I made an example and didn't hook it up lol